test/unit/components/VTextField/VTextField.spec.js

```diff
@@ -10,6 +10,17 @@ test('VTextField.js', ({ mount }) => {
     expect(wrapper.html()).toMatchSnapshot()
   })
 
+  it('should pass required attr to the input', () => {
+    const wrapper = mount(VTextField, {
+      attrs: {
+        required: true
+      }
+    })
+
+    const input = wrapper.find('input')[0]
+    expect(input.getAttribute('required')).toBe('required')
+  })
+
```

Comment:
Is a test really needed for this? It's basically impossible for it to regress.

Reply:
Or unless there is another refactor in the future. I don't think that having test for something that is unlikely (but not impossible) to regress is worse than not having it